src/js/components/unspent-transaction-output-view.jsx

```diff
@@ -46,6 +48,9 @@ class UnspentTransactionOutputView extends Component {
 
     componentWillUnmount() {
         clearInterval(this.updaterHandler);
+        if (this.state.datatable_loading) {
+            API.interruptTransaction().then(_ => _);
```

Comment:
remote it, it is interrupting send of a payment (transaction), not just any ajax